### ISSUE TITLE
Password alignment setting does not get respected

### ISSUE THREAD

**Issue by pioneer** [prikhi/lightdm-mini-greeter#58]
It's always the right alignment on my Manjaro/Cinnamon. I can see from the code the right is the default but seems it's always. Here is an excerpt from my config:

```
[greeter]
#...
# The text alignment for the password input. Possible values are:
# "left" or "right"
password-alignment = left
```

Any ideas how can I debug this issue or how it can be fixed?

**Comment by pioneer** [comment]
I have succeeded by adding 0 as a default alignment instead of 1 in the `config.c` file, and then recompiling the code, but of course it should be sorted out in a more proper way.

**Comment by prikhi** [comment]
Hi, just re-tested this on Arch Linux and it seems to be fine. What Linux distribution are you using? Can you report your GTK version as well?

**Comment by prikhi** [comment]
Ah, just noticed you said Manjaro/Cinnamon.

Can you do me a favor and see if you have a trailing space at the end of the `password-alignment` configuration line? That's an issue that people have been encountering recently & is an easy fix.

Meanwhile, I'll set up a Manjaro VM and test it out.

**Comment by prikhi** [comment]
Possibly fixed in dc1b4dfeb8d5971bfa21c462c6d44a586fcd704e.

Can you try the `lightdm-mini-greeter-git` package instead?

**Comment by pioneer** [comment]
No, I don't have a trailing space at that line. And, the `lightdm-mini-greeter` package got updated recently, so I don't have to install the `git` one anymore, I can confirm it's fixed now, thanks a lot!
